buildSrc/src/main/java/org/opensearch/gradle/Jdk.java

```diff
@@ -229,4 +230,15 @@ private void parseVersion(String version) {
         hash = jdkVersionMatcher.group(5);
     }
 
+    private String translateJdkArchitecture(String architecture) {
```

Comment:
Do u want to change to return architecture == "arm64"? "aarch64" : architecture; ? thx

Reply:
I like that, sure.